src/NexusMods.App.UI/Controls/GameWidget/GameWidget.axaml

```diff
@@ -19,51 +19,60 @@
     </Design.DataContext>
 
     <Border
-        Height="228"
-        Width="180"
+        Height="293"
+        Width="184"
         x:Name="GameWidgetBorder">
-        <Border x:Name="MainBorder"
-            Height="228"
-            Width="180">
-            <Grid RowDefinitions="106, *">
-                <Image
-                    Grid.Row="0"
-                    Grid.RowSpan="2"
-                    Height="228"
-                    Stretch="UniformToFill"
-                    Width="180"
-                    x:Name="BlurryImage" />
-                <Image
-                    Grid.Row="0"
-                    Width="180"
-                    x:Name="GameImage" />
-                <Border Grid.Row="1">
-                    <Grid RowDefinitions="Auto, 1,  *"
-                          Margin="16,0,16,0"
-                          VerticalAlignment="Top"
-                          Width="148">
-                        <TextBlock
-                            Grid.Row="0"
-                            Height="34"
-                            Margin="0,8"
-                            VerticalAlignment="Center"
-                            TextTrimming="CharacterEllipsis"
-                            TextWrapping="Wrap"
-                            x:Name="NameTextBlock" />
-                        <Border
-                            BorderThickness="0,0,0,1"
-                            Grid.Row="1"
-                            Margin="0"
-                            x:Name="SeparationLine"/>
-                        
-                        <!-- DetectedGame -->
+        
+            <StackPanel Spacing="12">
+                
+                <!-- image section -->
+                
+                <!-- using a grid so we can layer elements -->
+                <Border x:Name="ImageSectionBorder">
+                    <Grid RowDefinitions="Auto" Height="184" x:Name="ImageGrid">
+                        <Image Grid.Row="0"
+                               x:Name="GameImage"
+                        />
+                        <Border Grid.Row="0"
+                                Height="36"
```

Comment:
What do you mean with "layer elements" here?

Reply:
I will remove the comment, it was just to remind me that the grid is the easiest way to have elements over the top of other elements